code/modules/reagents/reagents/toxins.dm

```diff
@@ -438,6 +456,7 @@
 	taste_description = "sludge"
 	reagent_state = LIQUID
 	color = "#13BC5E"
+	nerve_system_accumulations = 0
```

Comment:
```suggestion
	nerve_system_accumulations = 50
```
Theoretically transforming you into a slime fucks up your entire bodily structure...